### PR TITLE
FLOE-469

### PR DESCRIPTION
**Pull request by dayotte**
@jhung hope I've done this right - seems I may not have committed it properly?

### CODE REVIEW

File: src/documents/css/idg.css.styl [hunk 1]
```diff
@@ -425,17 +425,23 @@ contrast-theme(foregroundColor, backgroundColor) {
...
         }
+        
         svg {
             fill: foregroundColor;
             background-color: backgroundColor !important;
         }
     }
+    
+    .docs-template-topics-active {
+          border-left: 0.3rem solid foregroundColor;
```
Comment (jhung) [inline src/documents/css/idg.css.styl]: You can take this line (#422) and move it to line 428. You can then erase lines 442 and 444.

Comment (dayotte) [inline src/documents/css/idg.css.styl]: you mean #442, right? move 442 to 428?

Comment (dayotte) [inline src/documents/css/idg.css.styl]: and then delete which lines?

Comment (jhung) [inline src/documents/css/idg.css.styl]: Ignore what I said. I was reading too quickly. Everything looks good - I've merged.

**Pull request merged**